core-services/src/main/resources/schema_metadata.json

```diff
@@ -34,6 +34,9 @@
       },
       "flow_id": {
         "type": "string"
+      },
+      "published_by": {
+        "type": "string"
```

Comment:
I think it should be also included for `compact` and `compact_and_delete`

Reply:
done